Source/SwiftLintFramework/Rules/VariableNameRule.swift

```diff
@@ -42,19 +54,28 @@ public struct VariableNameRule: ASTRule {
     public func validateFile(file: File, kind: SwiftDeclarationKind,
                              dictionary: XPCDictionary) -> [StyleViolation] {
         return file.validateVariableName(dictionary, kind: kind).map { name, offset in
-            let nameCharacterSet = NSCharacterSet(charactersInString: name)
-            let description = self.dynamicType.description
-            let location = Location(file: file, byteOffset: offset)
-            if !NSCharacterSet.alphanumericCharacterSet().isSupersetOfSet(nameCharacterSet) {
-                return [StyleViolation(ruleDescription: description,
-                    severity: .Error,
-                    location: location,
-                    reason: "Variable name should only contain alphanumeric characters: '\(name)'")]
-            } else if kind != SwiftDeclarationKind.VarStatic && nameIsViolatingCase(name) {
-                return [StyleViolation(ruleDescription: description,
-                    severity: .Error,
-                    location: location,
-                    reason: "Variable name should start with a lowercase character: '\(name)'")]
+            if !config.excluded.contains(name) {
```

Comment:
Perfect use-case for a guard here as well as in `VariableNameRule`. I always forget about those. :)